src/main/groovy/jenkins/automation/utils/CommonUtils.groovy

```diff
@@ -19,4 +24,31 @@ class CommonUtils {
             }
         }
     }
+
+/**
+ * Utility function to add extended email
+ *
+ * @param triggers List<String> triggers E.g Failure, Fixed etc...
+ * @param String emails Comma separated string of emails
+ * @param sendToDevelopers Default false,
+ * @param includeCulprits Default false,
+ * @param sendToRecipientList Default true
+ *
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#common-utils" target="_blank">BDD job Example</a>
+ */
+
+    static void extendedEmail(context, List<String> triggers = ["Failure", "Fixed"], String emails, sendToDevelopers = false, includeCulprits = false, sendToRecipientList = true) {
```

Comment:
The extendedEmail block takes a string http://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.helpers.publisher.PublisherContext.extendedEmail and I am a little hesitant to diverge from that. I can still make it work for the builder doing cool groovy things http://grails.asia/groovy-list-to-string inside the builders themselves. **OR** I can override the function to also take in a list with a signature like so: +    static void extendedEmail(context, List<String> triggers = ["Failure", "Fixed"], List emails, sendToDevelopers = false, includeCulprits = false, sendToRecipientList = true)